### ISSUE TITLE
Having the official Ruffle extension installed breaks the Shift + E shortcut

### ISSUE THREAD

**Issue by luni3359** [danbooru/danbooru#4783]
Just as the title reads, there's some exception thrown in the console that somehow manages to break the `Shift + E` shortcut. I haven't noticed any other shortcuts breaking thus far, but I haven't tested. I can only say that it's definitely not the shift key because `Shift + F` works.

This happened on Firefox. There don't seem to be any problems on Chromium.

**Comment by Dankbooru** [comment]
Also experienced this issue. Have resorted to disable the extension for now.

**Comment by evazion** [comment]
I can't reproduce this.

**Comment by nonamethanks** [comment]
This has been fixed by d441739ab2d2eb6674f5a989cd34d7e0fc83fccd. I used to have the same problem but now it doesn't happen anymore.

**Comment by nonamethanks** [comment]
Spoke too soon. It works in flash posts, but not in non-flash posts.

Here's the full trace (tested on https://danbooru.donmai.us/posts/4749304 with the latest ruffle for chrome at https://ruffle.rs/#downloads):
```
4749304:7 Uncaught TypeError: Cannot read properties of undefined (reading 'appendChild')
    at Object.initialize_ruffle_player (application-237cebbe071cb34ce095.js:1)
    at Object.initialize_all (application-237cebbe071cb34ce095.js:1)
    at HTMLDocument.<anonymous> (application-237cebbe071cb34ce095.js:1)
    at f (857-ff5115a2d6ceb23b4291.js:2)
    at p (857-ff5115a2d6ceb23b4291.js:2)
    at nrWrapper (4749304?q=status%3Aany:7)
```

**Comment by evazion** [comment]
Should be fixed in 440d2c8.
